lua/lualine/components/buffers/buffer.lua

```diff
@@ -38,12 +38,94 @@ function Buffer:get_props()
     local status, _ = pcall(require, 'nvim-web-devicons')
     if not status then
       dev, _ = '', ''
+    elseif self.filetype == 'minideps-confirm' then
```

Comment:
Can you put a comment at the start of these and at the end of these. stating the icons in this block exist in mini.icons (with link). and that it's relying on the fact that mini.icons patching nvim-web-devicons interface. Tbh this feels like an anti-pattern that they are patching other modules like this. I'm still not sure about relying on it like this. especially not without proper documentation.

Reply:
Maybe you are right, these adjustments can wait until the subsequent development of `nvim-web-devicons` or is `lualine` directly compatible with `mini.icons`?